app/models/policy_area.rb

```diff
@@ -6,4 +6,6 @@ class PolicyArea < ActiveRecord::Base
   validates :name, presence: true, uniqueness: true
   validates :slug, presence: true, uniqueness: true
   validates :description, presence: true
+
+  serialize :organisation_content_ids, Array
```

Comment:
Are we able to use some of postgres's functionality here? http://www.postgresql.org/docs/9.1/static/arrays.html